### DISCUSSION TITLE
termux nmap --iflist empty

### DISCUSSION THREAD

**Discussion by mishaparem** [termux/termux-packages#9000] [Q&A] [answered]
Hello all!

I am trying to connect to my home computer. I've set up openvpn, seems like the connection is working fine, but I can't nmap my network from termux (to find my computer ip address). I can see the vpn tunnel:
ifconfg:
tun0: x.x.x.x 255.255.255.252

but when I run nmap -A x.x.x.x/30 -e tun0 I get:
I cannot figure out what source address to use for device tun0, does it even exist?

I tried nmap --iflist, but it's empty:
INTERFACES: NONE FOUND(!)
ROUTES:  NONE FOUND(!)

same with nmap --iflist --unprivileged

Any help, ideas?

**Comment by ar664** [comment, marked as answer]
Termux can't bind to specific interfaces due to permissions. You can also see this when you try to ping through one
```
~ $ ping -I eth0 1.1.1.1
ping: SO_BINDTODEVICE: Operation not permitted
```

**Reply by sylirre** [reply]
Right, but there probably more issues. Latest Android versions (since 10?) do not allow accessing `/proc/net`, so accessing network info will require root permissions.
